modules/core/include/opencv2/core/mat.hpp

```diff
@@ -3588,6 +3588,9 @@ class CV_EXPORTS MatExpr
     Scalar s;
 };
 
+void checkOperandsExist(const Mat& a);
+void checkOperandsExist(const Mat& a, const Mat& b);
+
```

Comment:
These functions should not be a part of public OpenCV API.

Moreover, they are "local", so you should use "static" before definition instead.

Reply:
Thanks @alalek. I forgot about that. 

I suppose then this is best: Remove declarations from `mat.hpp` **and** add `static` qualifier to _checkOperandsExist_ in `matrix_expressions.cpp`. So _checkOperandsExist_ will _not_ be exposed outside `matrix_expressions.cpp`.

Reply:
Right!